operators/multiclusterobservability/main.go

```diff
@@ -302,10 +305,11 @@ func main() {
 		setupLog.Error(err, "unable to set up ready check")
 		os.Exit(1)
 	}
-	if err := operatorsutil.RegisterDebugEndpoint(mgr.AddMetricsExtraHandler); err != nil {
-		setupLog.Error(err, "unable to set up debug handler")
-		os.Exit(1)
-	}
+	// TODO(saswatamcode): Address this later.
+	// if err := operatorsutil.RegisterDebugEndpoint(mgr.AddMetricsExtraHandler); err != nil {
```

Comment:
Probably needs refactoring by calling `mgr.Add` and passing servers which listen on given path.